### PR TITLE
Creating .env.sample file and updating README.md according to it.

### LINKED ISSUE #893: Add a .env.sample for the Netmanager
**Is your feature request related to a problem? Please describe.**
While setting up the netmanager, it's difficult to know all the environmental variables one would need to run the application 

**Describe the solution you'd like**
- Can we have a `.env.sample` file as a template for the `.env` file at the root of the [netmanager folder](
https://github.com/airqo-platform/AirQo-frontend/tree/staging/netmanager) with a list of environmental variables and dummy values in it.
- Also, can we update the [netmanager README](https://github.com/airqo-platform/AirQo-frontend/blob/staging/netmanager/README.md) to let users know that the `.env.sample` is merely a template they need to fill with actual values when creating their respective local `.env` file.

**Additional context**
- The contents of the `.env.sample` may include:
```
REACT_APP_BASE_URL=https://staging-platform.airqo.net/api/v1/
REACT_APP_BASE_ANALYTICS_URL=
REACT_APP_BASE_AUTH_SERVICE_URL=
REACT_APP_BASE_DATA_MANAGEMENT_URL=
REACT_APP_BASE_DEVICE_MONITORING_URL=
REACT_APP_BASE_DEVICE_REGISTRY_URL= 
REACT_APP_BASE_LOCATE_URL=
REACT_APP_BASE_LOCATION_REGISTRY_URL=
REACT_APP_BASE_PREDICT_URL=
REACT_APP_MAX_CONFIDENCE_INTERVAL=
REACT_APP_CLOUDINARY_NAME=abcdef
REACT_APP_CLOUDINARY_PRESET=abcdef
REACT_APP_MAPBOX_TOKEN=pk.ey123abc
REACT_APP_JIRA_KEY=abcdef
NODE_PATH=src/
PORT=5000
```
- Empty values are not required for the application to run
- [Here](https://github.com/airqo-platform/AirQo-frontend/blob/staging/calibrate/.env.sample) is a `.env.sample` example

### PR DESCRIPTION
**Pull request by rohitkori**
#### Summary of Changes (What does this PR do?)
- Closes #893 

#### Is this change ready to hit production in its current state?
- [ ] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [ ] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- #893 

#### Screenshots (optional)

### CODE REVIEW

**Review by 123MwanjeMike — CHANGES_REQUESTED** [review]
Good work @rohitkori.
However, these changes need to be implemented in the [netmanager folder](https://github.com/airqo-platform/AirQo-frontend/tree/staging/netmanager). Could you make that change so it's all good to review and merge.
When ready, check the respective boxes in the PR description for reviewers to know the readiness state of the PR.
Thank you.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ This repo is made up of six (6) product sub directories:
 - [Netmanager](/netmanager/)
 - [Website](/website/)
 
+## .env.sample file
+
+You need to fill actual values to the environment variables while creating the respective `.env` file from `.env.sample` file
```
[the remark this answers is not in the document]
Comment (rohitkori) [reply README.md:22]: okk sure

[the remark this answers is not in the document]
Comment (rohitkori) [reply README.md]: I had done the change specified by you. Please review it.

[the remark this answers is not in the document]
Comment (123MwanjeMike) [reply README.md]: Thank you @rohitkori 
However, the `.env.sample` file is supposed to be added the netmanager folder(https://github.com/rohitkori/AirQo-frontend/tree/staging/netmanager). Also, the changes in the README are supposed to be made to the netmanager README(https://github.com/rohitkori/AirQo-frontend/blob/staging/netmanager/README.md) not the one at repository root(https://github.com/rohitkori/AirQo-frontend/blob/staging/netmanager/README.md).

Please make make this migration and it'll be perfect.

File: README.md [hunk 2]
```diff
@@ -17,6 +17,10 @@ This repo is made up of six (6) product sub directories:
 - [Netmanager](/netmanager/)
 - [Website](/website/)
 
+## .env.sample file
+
```
Comment (123MwanjeMike) [inline README.md:21]: Please move this change to this line of the netmanager README https://github.com/rohitkori/AirQo-frontend/blob/bd7cab7856e425c5722e650a363eea310a91dc78/netmanager/README.md?plain=1#L9

**Review by 123MwanjeMike — COMMENTED** [review]
Thank you @rohitkori 
Just a guiding comment

**Comment by 123MwanjeMike** [comment]
Implementation of this PR is carried to #917
cc @rohitkori

**Pull request closed without merging**